quinn-proto/src/connection/mod.rs

```diff
@@ -2943,6 +3005,20 @@ where
         self.path.sending_ecn
     }
 
+    /// Mock updating retire_prior_to field in NEW_CONNECTION_ID frame
+    #[cfg(test)]
+    pub(crate) fn mock_retire_cid(&mut self, v: u64, n: u64) {
```

Comment:
I will change to `rotate_rem_cids` as it is `remote` cid that this function actually changes. Otherwise i am OK with the change. Will also add the assert there.